docs/GettingStarted.md

```diff
@@ -28,8 +28,8 @@ To contribute, check out the [contribution guidelines][contribution docs] and [d
  - [Installation](#installation)
      - [Setup the Datadog Agent for tracing](#setup-the-datadog-agent-for-tracing)
      - [Instrument your application](#instrument-your-application)
-        - [Rails applications](#rails-applications)
-        - [Ruby applications](#ruby-applications)
+        - [Rails or Hanami applications](#rails-or-hanami-applications)
```

Comment:
Should we include Sinatra as well?

Reply:
Sinatra does not have the same framework support because we do not have a reliable instrumentation point that happens "after all gems have been required by the host application".
Both Rails and Hanami have such callback.

Reply:
Thanks for the explanation